### PR TITLE
add space API json file

### PR DESCRIPTION
**Pull request by alifeee**
please see -> https://spaceapi.io/

> [SpaceAPI is] A standardized JSON file that can be used to expose information about hackerspaces, makerspaces, fablabs, chaostreffs and the like across the world to web apps or any other application. 

for now it is just a static file with information about the space

it would be neat if it was active and had live-data about [space open-ness (either calendar-based or connected to the MQTT door system)] / [webcams] / [sensors] / [events], as many other hackspaces do (see <https://api.fablab-nuernberg.de/spaceapi.php> e.g., or many others on <https://directory.spaceapi.io/>), but previous discussions about open data make me think that this idea would not be well adopted. So, for now, it is static and not networked.

after this PR, one should be made to add the URL to the directory on <https://github.com/SpaceApi/directory>

thanks

### CODE REVIEW

**Comment by alifeee** [comment]
see what it would look like on this test site -> <https://api.fablab-nuernberg.de/spaceapi.php>

**Review by LouiseMcMahon — CHANGES_REQUESTED** [review]
Can you generate the file from the _config.my and add any bits that you don’t have that are missing to the config or a data file.

Could you also check that anything you add to the config isn’t duplicated else where and also uses the confg.

I can just see this file getting forgotten about and then not being updated.

**Comment by alifeee** [comment]
yes, that's a good idea. I agree.

I moved everything bar the link to the wiki, as it didn't seem incredibly apt. To be honest, I could argue for just removing it. I may do so now.

File: _config.yml [hunk 1]
```diff
@@ -18,18 +18,44 @@
...
+# information - mainly used in spaceapi.json
+address: "Sheffield Hackspace, Portland Works, Randall St, Highfield, Sheffield S2 4SJ"
+latitude: 53.369933
+longitude: -1.473812
+timezone: "Europe/London"
+
+# membership
+membership_plans:
+  - name: Normal membership
+    value: 6
```
Comment (LouiseMcMahon) [inline _config.yml:50]: Could we pull these into the joining page? Doesn't need to be complex. Just so if someone goes to update prices they wont miss one place or the other.

File: _config.yml [hunk 2]
```diff
@@ -18,18 +18,44 @@
...
 twitter_username: "@shhmakers"
 github_username:  "sheffieldhackspace"
 facebook_username: "SHHMakers"
 instagram_username: "SHHMakers"
-logo: /assets/images/logo.svg
-goat_counter_url: "https://sheffieldhackspacewebsite.goatcounter.com"
+
+# information - mainly used in spaceapi.json
+address: "Sheffield Hackspace, Portland Works, Randall St, Highfield, Sheffield S2 4SJ"
+latitude: 53.369933
```
Comment (LouiseMcMahon) [inline _config.yml:43]: Can we use these in the osm.html file please

**Review by LouiseMcMahon — CHANGES_REQUESTED** [review]

**Comment by alifeee** [comment]
I have done so

see on <https://shhm-spaceapi-preview.surge.sh/joining/>

image

**Review by LouiseMcMahon — APPROVED** [review]

**Comment by alifeee** [comment]
lovely thank you -> https://github.com/SpaceApi/directory/pull/269

**Pull request merged**